src/scripts/main.js

```diff
@@ -1 +1,98 @@
 'use strict';
+
+const createMessageElement = () => {
+  const message = document.createElement('div');
+
+  message.setAttribute('data-qa', 'notification');
+
+  return message;
+};
+
+// First promise
+const firstMessage = createMessageElement();
+const firstPromise = new Promise((resolve, reject) => {
+  let clicked = false;
+
+  document.addEventListener('click', () => {
+    clicked = true;
+
+    const resolvedValue = 'First promise was resolved';
+
+    resolve(resolvedValue);
+  });
+
+  setTimeout(() => {
+    if (!clicked) {
+      const rejectedValue = 'First promise was rejected';
+
+      reject(rejectedValue);
+    }
+  }, 3000);
+});
+
+firstPromise
+  .then((resolvedValue) => {
+    firstMessage.classList.add('success');
+    firstMessage.innerText = resolvedValue;
+
+    document.body.append(firstMessage);
+  })
+  .catch((rejectedValue) => {
+    firstMessage.classList.add('error');
+    firstMessage.innerText = rejectedValue;
+
+    document.body.append(firstMessage);
```

Comment:
you can create that firstMessage here, in the then or catch

Reply:
or create some helper function and move all this logic there because we can see this logic almost the same